examples/selfhosted/main.c

```diff
@@ -577,8 +584,9 @@ wrap_ecdsa_verify(unsigned int n, unsigned int i)
         hash[x] = ee_rand();
     }
     slen = 256; // Note: this is also an input to ee_ecdsa_sign
-    sig  = (unsigned char *)th_malloc(slen); // should be 71, 72 B
```

Comment:
Is this comment incorrect? I thought the signature could only be one of two sizes?

Reply:
I thought the signature would be 64 (0x40) bytes long.